extensions/wikia/Search/classes/Services/QuestDetails/QuestDetailsSolrHelper.class.php

```diff
@@ -251,60 +262,63 @@ protected function getRevision( $item ) {
 	}
 
 	protected function getCommentsNumber( $item ) {
-		$titles = Title::newFromIDs( $item[ 'pageid' ] );
-		if( empty( $titles ) ) {
+		$titles = Title::newFromIDs( $item['pageid'] );
+		if ( empty( $titles ) ) {
 			return null;
 		}
-		$title = $titles[ 0 ];
+		$title = $titles[0];
 		if ( class_exists( 'ArticleCommentList' ) ) {
 			$commentsList = ArticleCommentList::newFromTitle( $title );
+
 			return $commentsList->getCountAllNested();
 		}
+
 		return null;
 	}
 
 	protected function addThumbnailsInfo( &$result ) {
-		$articleIds = [ ];
+		$articleIds = [];
 		foreach ( $result as &$item ) {
-			$articleIds[ ] = $item[ 'id' ];
+			$articleIds[] = $item['id'];
 		}
 
 		$thumbnails = $this->getArticlesThumbnails( $articleIds );
 
 		foreach ( $result as &$item ) {
-			$id = $item[ 'id' ];
-			$thumbnailProps = $thumbnails[ $id ];
+			$id = $item['id'];
+			$thumbnailProps = $thumbnails[$id];
 			foreach ( $thumbnailProps as $key => $value ) {
-				if( !empty( $value ) ) {
-					$item[ $key ] = $value;
+				if ( !empty( $value ) ) {
+					$item[$key] = $value;
 				}
 			}
 		}
 	}
 
 	protected function getArticlesThumbnails( $articles ) {
 		$ids = !is_array( $articles ) ? [ $articles ] : $articles;
-		$result = [ ];
+		$result = [];
 		if ( self::DEFAULT_THUMBNAIL_WIDTH > 0 && self::DEFAULT_THUMBNAIL_HEIGHT > 0 ) {
 			$is = $this->getImageServing( $ids, self::DEFAULT_THUMBNAIL_WIDTH, self::DEFAULT_THUMBNAIL_HEIGHT );
 			//only one image max is returned
 			$images = $is->getImages( 1 );
 			//parse results
 			foreach ( $ids as $id ) {
 				$data = [ 'thumbnail' => null, 'original_dimensions' => null ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "original_dimensions" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)